code/src/components/Albums.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { AlbumCover } from './AlbumCover.js'
+import { AlbumData } from './AlbumData.js'
+import './album.css'
+
+export const Albums = (props) => {
+    return (
+        <div className="albumCard">
+            <AlbumCover
+                key={props.album.images[0].url}
```

Comment:
You don't need a key here (or below). You only need keys when you're in a `.map()`